packages/apps/shopify-api/future/flags.ts

```diff
@@ -36,10 +42,15 @@ export function logDisabledFutureFlags(
   const logFlag = (flag: string, message: string) =>
     logger.info(`Future flag ${flag} is disabled.\n\n  ${message}\n`);
 
-  if (!config.future?.v10_lineItemBilling) {
+  if (!config.future?.v11_lineItemBilling) {
     logFlag(
-      'v10_lineItemBilling',
+      'v11_lineItemBilling',
       'Enable this flag to use the new billing API, that supports multiple line items per plan.',
     );
   }
+  if (config.future?.v10_lineItemBilling) {
+    logger.info(
```

Comment:
Should we use info or debug here?

Reply:
Maybe `deprecated` set to v11? Also worth pointing out that if we're not calling `logFlag` here we should include the flag name in the message :)